metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -2140,6 +2144,21 @@ def parse_custom_sql(
             schema_aware=not self.config.sql_parsing_disable_schema_awareness,
         )
 
+        assert parsed_result is not None
+
+        if parsed_result.debug_info.table_error:
+            logger.warning(
+                f"Failed to extract table lineage from datasource {datasource_urn}: {parsed_result.debug_info.table_error}"
+            )
+            self.report.num_upstream_table_lineage_failed_parse_sql += 1
+        if parsed_result.debug_info.column_error:
```

Comment:
```suggestion
        elif parsed_result.debug_info.column_error:
```

Reply:
technically this is redundant, but it makes the code slightly clearer

Reply:
https://github.com/datahub-project/datahub/blob/48b5a6221c8a203268e8905de3238d9d47411f75/metadata-ingestion/src/datahub/sql_parsing/sqlglot_lineage.py#L153-L154

both fields are optional and so both fields could be present 😅
although very unlikely, that's why I implemented with two separated `if` statements

I'm taking your suggestion anyway